lib/identity_doc_auth/error_generator.rb

```diff
@@ -36,7 +36,7 @@ def initialize(config)
       'Control Number Crosscheck': { type: BACK, msg_key: Errors::CONTROL_NUMBER_CHECK },
       'Document Classification': { type: ID, msg_key: Errors::ID_NOT_RECOGNIZED },
       'Document Crosscheck Aggregation': { type: ID, msg_key: Errors::DOC_CROSSCHECK },
-      'Document Expired': { type: ID, msg_key: Errors::EXPIRATION_CHECKS },
+      'Document Expired': { type: ID, msg_key: Errors::DOCUMENT_EXPIRED },
```

Comment:
Suggestion for naming consistency
```suggestion
      'Document Expired': { type: ID, msg_key: Errors::DOCUMENT_EXPIRED_CHECK },
```